string node_name
string config_name
string new_cnofig
---
string result
